jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -826,54 +812,141 @@ def _make_div_rem_harness(name, *, shapes=((2,), (2,)), dtype=np.float32,
   ]
 )
 
-_LAX_BINARY_ELEMENTWISE = (
-  lax.add, lax.atan2, lax.div, lax.igamma, lax.igammac, lax.max, lax.min,
```

Comment:
Are there all the binary ops tested? I did not see below the harnesses for add and and sub. I think that div/rem and min/max are already tested separately.

Reply:
Yes! `add` was already tested in `lax_add_mul` (line 721) and `sub` in `lax_sub` (line 516).

Reply:
I see. Those don't have broadcasting. I wonder if it is important to add broadcasting checks everywhere. Maybe not.